api/src/main/java/org/openmrs/Diagnosis.java

```diff
@@ -0,0 +1,124 @@
+package org.openmrs;
```

Comment:
@esirK Kindly add the license header in this file and any other file it is not in

Reply:
Sure. On it.